wx_test_output.hpp

```diff
@@ -57,9 +57,9 @@ class output_file_existence_checker
         return fs::exists(path_);
         }
 
-    std::string const& path() const
+    fs::path const& path() const
         {
-        return path_.string();
+        return path_;
         }
```

Comment:
It looks like this is only used in a single place (`create_open_mec()`) to output the path to a stream. I wonder if we should just define `operator<<()` instead...